### PR TITLE
create NG911 GIS Roadmap post

### PR DESCRIPTION
**Pull request by gregbunce**
Pull request description
------------------------

### CODE REVIEW

**Comment by gregbunce** [comment]
Thanks

File: _posts/2018-08-31-utah-ng911-gis-roadmap-v1.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+date: 2018-08-31 11:00:00
+categories:
+  - Featured
+tags:
+  - '911'
+  - next generation 911
+  - utah 911
+---
+
+As the State of Utah moves closer to Next-Generation-911 implementation (NG911), GIS data providers will soon be required to adhere to national standards set forth by the [National Emergency Number Association (NENA)](https://www.nena.org/) and the [State's 911 Division](http://uca911.org/911-division) [(section 6 or ref 1)](#further-reading).  It can be daunting navigating these `standards` and `best practice` documents, so the intent of this article is to demystify the relevant information, and lay out a road map of where we are going.
```
Comment (atribolet) [inline _posts/2018-08-31-utah-ng911-gis-roadmap-v1.md]: Remove comma before "and lay out"

**Comment by gregbunce** [comment]
No worries. Let me know if I can be of any assistance.

**Comment by gregbunce** [comment]
this is now ready for prime time

**Comment by gregbunce** [comment]
thanks, @atribolet for your comments and edits!

**Comment by atribolet** [comment]
Whoo-hoo! :D You're so welcome!

**Comment by steveoh** [comment]
I will merge this but I am going to 

1. remove all references to gis.utah.gov in links
1. replace ugly images with [font-awesome icons](https://fontawesome.com/icons?d=gallery) **please use these as much as you want** instead of images
1. compress images

**Comment by atribolet** [comment]
Lol nice work! ;) And font-awesome icons? I haven't heard of that! Excited to learn more! Cool! Thanks, Steve! :)

**Comment by gregbunce** [comment]
cool, yo - noted.  gracias!

**Pull request closed without merging**